.github/workflows/configurations.yml

```diff
@@ -20,115 +22,68 @@ jobs:
-      - name: Push
-        uses: crossplane-contrib/xpkg-action@1696d9091687aad2895436cb5656faa956628b0f # v0.2.0
-        with:
-          command: push configuration -f cluster/packages/aws/getting-started-with-aws.xpkg registry.upbound.io/xp/getting-started-with-aws:${{ steps.tagger.outputs.VERSION_TAG }}
 
-  getting-started-with-aws-with-vpc:
-    runs-on: ubuntu-22.04
-    if: github.repository == 'crossplane/crossplane'
-    steps:
-      - name: Checkout
-        uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
-        with:
-          submodules: true
-          fetch-depth: 0
-      - name: Set tag
-        run: echo "VERSION_TAG=$(git describe --dirty --always --tags | sed 's/-/./2' | sed 's/-/./2' )" >> $GITHUB_OUTPUT
-        id: tagger
-      - name: Login to Docker
-        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d # v3
-        if: env.DOCKER_USR != ''
-        with:
-          registry: registry.upbound.io
-          username: ${{ secrets.UPBOUND_XP_ROBOT_USR }}
-          password: ${{ secrets.UPBOUND_XP_ROBOT_PSW }}
-      - name: Build
-        uses: crossplane-contrib/xpkg-action@1696d9091687aad2895436cb5656faa956628b0f # v0.2.0
+      - name: Setup Go
+        uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe # v4
         with:
-          channel: master
-          version: current
-          command: build configuration -f cluster/packages/aws-with-vpc --name=getting-started-with-aws-with-vpc.xpkg
-      - name: Push
-        uses: crossplane-contrib/xpkg-action@1696d9091687aad2895436cb5656faa956628b0f # v0.2.0
-        with:
-          command: push configuration -f cluster/packages/aws-with-vpc/getting-started-with-aws-with-vpc.xpkg registry.upbound.io/xp/getting-started-with-aws-with-vpc:${{ steps.tagger.outputs.VERSION_TAG }}
+          go-version: ${{ env.GO_VERSION }}
+
+      - name: Vendor Dependencies
+        run: make vendor vendor.check
 
-  getting-started-with-gcp:
+      - name: Build Go Binaries
+        run: make -j2 go.build
+      
+      - name: Upload crossplane cli
+        uses: actions/upload-artifact@a8a3f3ad30e3422c9c7b888a15615d19a852ae32 # v3
+        with:
+          path: _output/bin/linux_amd64/crank
+          name: crossplane-cli
+          retention-days: 1
+
+  configs:
     runs-on: ubuntu-22.04
     if: github.repository == 'crossplane/crossplane'
+    needs: build-crossplane-cli
+    strategy:
+      fail-fast: false
+      matrix:
+        configuration:
+          - { name: "getting-started-with-aws", path: "cluster/packages/aws/getting-started-aws.xpkg" }
+          - { name: "getting-started-with-aws-with-vpc", path: "cluster/packages/aws-with-vpc/getting-started-with-aws-with-vpc.xpkg" }
+          - { name: "getting-started-with-gcp", path: "cluster/packages/gcp/getting-started-with-gcp.xpkg" }
+          - { name: "getting-started-with-azure", path: "cluster/packages/azure/getting-started-with-gcp.xpkg" }
```

Comment:
CC @plumbis to confirm, but I don't think these are actually used anymore. They were part of our documentation before the new docs.crossplane.io site. If they're unused we could probably delete them.

Reply:
this would be even better, let me know if we can remove them @plumbis, if that's the case, should we drop them also from the marketplace?

Reply:
None of these are in the docs. There are references to the older `snippets` docs directory but that stuff is all contained inside the docs. 

Docs has no dependencies on this.

Reply:
So I'm going to drop this action and remove the steps from the release procedure 👍

Reply:
Removed in a separate commit, so if we ever want to switch back the new job will still be in the history.